appinfo/info.xml

```diff
@@ -20,7 +20,7 @@ To whitelist `127.0.0.0/24` and also `192.168.0.0/24`:
 
 - `occ config:app:set limit_login_to_ip whitelisted.ranges --value 127.0.0.0/24,192.168.0.0/24`
 ]]></description>
-	<version>2.0.0</version>
+	<version>2.1.0</version>
```

Comment:
strictly speaking removing support for a major version of Nextcloud is a breaking change IMO, thus this should be a major version bump if the version in this app are semantical :)

Reply:
Yeah agreed

Reply:
Good for me :)

Reply:
Just so you know, when we update all of our shipped apps, we only bump minors (Joas), so maybe we should stop? I really have no strong feeling with this

Reply:
@nickvergessen fix it :P

Reply:
Sure, let me try to remember that next time